src/index.ts

```diff
@@ -365,4 +365,10 @@ export const createProbot = (options: Options) => {
 
 export type ApplicationFunction = (app: Application) => void;
 
-export { Logger, Context, Application, ProbotOctokit };
+export {
+  Logger,
+  Context,
+  Application,
+  ProbotOctokit,
+  WebhookPayloadWithRepository,
+};
```

Comment:
I'd like to make more explicit that we don't consider the `WebhookPayloadWithRepository` export part of the official API

```suggestion
export {
  Logger,
  Context,
  Application,
  ProbotOctokit,
};

// NOTE: exported types might change at any point in time
export {
  WebhookPayloadWithRepository
}
```